node/utils.ts

```diff
@@ -1,13 +1,17 @@
 import { Binding, TenantClient } from '@vtex/api'
 import { any, startsWith } from 'ramda'
+import { GENERATE_SITEMAP_EVENT } from './middlewares/generateMiddlewares/utils'
 
 export const CONFIG_BUCKET = 'configuration'
 export const CONFIG_FILE = 'config.json'
+export const GENERATION_CONFIG_FILE = 'generation.json'
 
 export const TENANT_CACHE_TTL_S = 60 * 10
 
 export const STORE_PRODUCT = 'vtex-storefront'
 
+const oneHourFromNowMS = () => `${new Date(Date.now() + 1 * 60 * 60 * 1000)}`
```

Comment:
do you need it to be a string?

```suggestion
const oneHourFromNowMS = () => new Date(Date.now() + 1 * 60 * 60 * 1000)
```